components/Canvas.js

```diff
@@ -31,7 +31,7 @@ export default function Canvas({
                 animate={animate}
                 id={item.id}
               >
-                {item.id}
+                {/* {item.id} */}
```

Comment:
Please delete code you don't need.

Reply:
That specific line displays the id of the container for level design. I just comment it out for the deployment, it will be deleted when the project is finished